### ISSUE TITLE
Select: In case of Filter and AutoFocusFilter enabled focus is lost

### ISSUE THREAD

**Issue by Sandip124** [primefaces/primevue#6194]
### Describe the bug

When enabling filter and autoFilterFocus on primevue select , then no options are selected (may be due to missing props auto item select just like in AutoComplete) and just closes and thus focus is lost

### Reproducer

https://stackblitz.com/edit/ty7yqi?file=src%2FApp.vue

### PrimeVue version

3.X.X and 4.X.X

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vue CLI App

### Steps to reproduce the behavior

1. Focus on select box, that should automatically focus on the filter and without typing anything and selecting any of this options, just hitting enter button will loose the focus.

### Expected behavior

If we dont use filter then focus is working just fine, and also with Auto Filter Focus enable or not enabled and importantly filter enabled,  shouldnot loose focus from the select.
